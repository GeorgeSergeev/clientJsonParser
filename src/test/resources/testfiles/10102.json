{
  "clientId": 10102,
  "subscribers": [{
	"id": 3000001,
	"spent": 1131
  } ,{
	"id": 3000002,
	"spent": 1132
  } ,{
	"id": 3000003,
	"spent": 1133
  } ,{
	"id": 3000004,
	"spent": 1134
  } ,{
	"id": 3000005,
	"spent": 1135
  } ,{
	"id": 3000006,
	"spent": 1136
  } ,{
	"id": 3000007,
	"spent": 1137
  } ,{
	"id": 3000008,
	"spent": 1138
  } ,{
	"id": 3000009,
	"spent": 1139
  } ,{
	"id": 30000010,
	"spent": 11310
  } ,{
	"id": 30000011,
	"spent": 11311
  } ,{
	"id": 30000012,
	"spent": 11312
  } ,{
	"id": 30000013,
	"spent": 11313
  } ,{
	"id": 30000014,
	"spent": 11314
  } ,{
	"id": 30000015,
	"spent": 11315
  } ,{
	"id": 30000016,
	"spent": 11316
  } ,{
	"id": 30000017,
	"spent": 11317
  } ,{
	"id": 30000018,
	"spent": 11318
  } ,{
	"id": 30000019,
	"spent": 11319
  } ,{
	"id": 30000020,
	"spent": 11320
  } ,{
	"id": 30000021,
	"spent": 11321
  } ,{
	"id": 30000022,
	"spent": 11322
  } ,{
	"id": 30000023,
	"spent": 11323
  } ,{
	"id": 30000024,
	"spent": 11324
  } ,{
	"id": 30000025,
	"spent": 11325
  } ,{
	"id": 30000026,
	"spent": 11326
  } ,{
	"id": 30000027,
	"spent": 11327
  } ,{
	"id": 30000028,
	"spent": 11328
  } ,{
	"id": 30000029,
	"spent": 11329
  } ,{
	"id": 30000030,
	"spent": 11330
  } ,{
	"id": 30000031,
	"spent": 11331
  } ,{
	"id": 30000032,
	"spent": 11332
  } ,{
	"id": 30000033,
	"spent": 11333
  } ,{
	"id": 30000034,
	"spent": 11334
  } ,{
	"id": 30000035,
	"spent": 11335
  } ,{
	"id": 30000036,
	"spent": 11336
  } ,{
	"id": 30000037,
	"spent": 11337
  } ,{
	"id": 30000038,
	"spent": 11338
  } ,{
	"id": 30000039,
	"spent": 11339
  } ,{
	"id": 30000040,
	"spent": 11340
  } ,{
	"id": 30000041,
	"spent": 11341
  } ,{
	"id": 30000042,
	"spent": 11342
  } ,{
	"id": 30000043,
	"spent": 11343
  } ,{
	"id": 30000044,
	"spent": 11344
  } ,{
	"id": 30000045,
	"spent": 11345
  } ,{
	"id": 30000046,
	"spent": 11346
  } ,{
	"id": 30000047,
	"spent": 11347
  } ,{
	"id": 30000048,
	"spent": 11348
  } ,{
	"id": 30000049,
	"spent": 11349
  } ,{
	"id": 30000050,
	"spent": 11350
  } ,{
	"id": 30000051,
	"spent": 11351
  } ,{
	"id": 30000052,
	"spent": 11352
  } ,{
	"id": 30000053,
	"spent": 11353
  } ,{
	"id": 30000054,
	"spent": 11354
  } ,{
	"id": 30000055,
	"spent": 11355
  } ,{
	"id": 30000056,
	"spent": 11356
  } ,{
	"id": 30000057,
	"spent": 11357
  } ,{
	"id": 30000058,
	"spent": 11358
  } ,{
	"id": 30000059,
	"spent": 11359
  } ,{
	"id": 30000060,
	"spent": 11360
  } ,{
	"id": 30000061,
	"spent": 11361
  } ,{
	"id": 30000062,
	"spent": 11362
  } ,{
	"id": 30000063,
	"spent": 11363
  } ,{
	"id": 30000064,
	"spent": 11364
  } ,{
	"id": 30000065,
	"spent": 11365
  } ,{
	"id": 30000066,
	"spent": 11366
  } ,{
	"id": 30000067,
	"spent": 11367
  } ,{
	"id": 30000068,
	"spent": 11368
  } ,{
	"id": 30000069,
	"spent": 11369
  } ,{
	"id": 30000070,
	"spent": 11370
  } ,{
	"id": 30000071,
	"spent": 11371
  } ,{
	"id": 30000072,
	"spent": 11372
  } ,{
	"id": 30000073,
	"spent": 11373
  } ,{
	"id": 30000074,
	"spent": 11374
  } ,{
	"id": 30000075,
	"spent": 11375
  } ,{
	"id": 30000076,
	"spent": 11376
  } ,{
	"id": 30000077,
	"spent": 11377
  } ,{
	"id": 30000078,
	"spent": 11378
  } ,{
	"id": 30000079,
	"spent": 11379
  } ,{
	"id": 30000080,
	"spent": 11380
  } ,{
	"id": 30000081,
	"spent": 11381
  } ,{
	"id": 30000082,
	"spent": 11382
  } ,{
	"id": 30000083,
	"spent": 11383
  } ,{
	"id": 30000084,
	"spent": 11384
  } ,{
	"id": 30000085,
	"spent": 11385
  } ,{
	"id": 30000086,
	"spent": 11386
  } ,{
	"id": 30000087,
	"spent": 11387
  } ,{
	"id": 30000088,
	"spent": 11388
  } ,{
	"id": 30000089,
	"spent": 11389
  } ,{
	"id": 30000090,
	"spent": 11390
  } ,{
	"id": 30000091,
	"spent": 11391
  } ,{
	"id": 30000092,
	"spent": 11392
  } ,{
	"id": 30000093,
	"spent": 11393
  } ,{
	"id": 30000094,
	"spent": 11394
  } ,{
	"id": 30000095,
	"spent": 11395
  } ,{
	"id": 30000096,
	"spent": 11396
  } ,{
	"id": 30000097,
	"spent": 11397
  } ,{
	"id": 30000098,
	"spent": 11398
  } ,{
	"id": 30000099,
	"spent": 11399
  } ,{
	"id": 300000100,
	"spent": 113100
  },{
	"id": 300000101,
	"spent": 1131
  }
  ]
}