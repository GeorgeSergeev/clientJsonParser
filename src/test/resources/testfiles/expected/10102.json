{
  "clientId" : 10102,
  "spentTotal" : 1146351,
  "isBig" : true
}